core/templates/pages/exploration-editor-page/services/exploration-property.service.spec.ts

```diff
@@ -51,11 +51,11 @@ describe('Exploration Property Service', () => {
 
   it('should create a new exploration properties object', function() {
     expect(function() {
-      explorationPropertyService.init('initial value');
+      explorationPropertyService.init('initial value' as unknown as ParamSpecs);
```

Comment:
Why this change? I don't see why an arbitrary exploration property would be "ParamSpecs".

Ditto below.

Reply:
Changed.